### PR TITLE
add prac 6 files

### PR DESCRIPTION
**Pull request by Lijiayi2**
@gzy20020626
@AdriantoHartono
@nora-zyy
@a-rosa
@JevanChndra

### CODE REVIEW

File: prac_06/car.py
Comment (AdriantoHartono) [inline prac_06/car.py:1]: The class already good written, but I think you need to add self.name in the __init__ function🙏

File: prac_06/guitar.py
Comment (AdriantoHartono) [inline prac_06/guitar.py:1]: The class code already good, but I think we don't need my_guitar and the output from print for this py file🙏

File: prac_06/guitar_test.py
Comment (AdriantoHartono) [inline prac_06/guitar_test.py:1]: It is already good, but I think you should import the class from the other file. So you don't need to write the class on the top of the code 🙏

File: prac_06/guitars.py
Comment (AdriantoHartono) [inline prac_06/guitars.py:1]: The code already work perfectly, but I think you missed to append some of the information (Gibson and Line guitars)

File: prac_06/languages.py
Comment (AdriantoHartono) [inline prac_06/languages.py:1]: The code already good, but I think you need to change the vb name become a better one, such as visual_basic maybe?😅

File: prac_06/programming_language.py
Comment (AdriantoHartono) [inline prac_06/programming_language.py:1]: The code already good written, but I think you don't need to run test on this file yet 😅

File: prac_06/used_cars.py
Comment (AdriantoHartono) [inline prac_06/used_cars.py:1]: I think the code has trouble, maybe when you import it from the prac_06.car, I think you forgot to put the name in the car.py🙏

File: prac_06/guitar_test.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# Test get_age() method
+print("Gibson L-5 CES get_age() - Expected 100. Got", guitar1.get_age())
+print("Another Guitar get_age() - Expected 9. Got", guitar2.get_age())
+
+# Test is_vintage() method
+print("Gibson L-5 CES is_vintage() - Expected True. Got", guitar1.is_vintage())
+print("Another Guitar is_vintage() - Expected False. Got", guitar2.is_vintage())
+
+# Test a wrong implementation of is_vintage() method
+print("50-year old guitar is_vintage() - Expected True. Got", guitar1.is_vintage())
```
Comment (a-rosa) [inline prac_06/guitar_test.py:29]: I think you are supposed to import the class from guitar.py instead of writing it

File: prac_06/guitars.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        year = input("Enter guitar year: ")
+        cost = input("Enter guitar cost: ")
+        guitar = Guitar(name, year, cost)
+        guitars.append(guitar)
+        print(f"{guitar.name} ({guitar.year}) : ${guitar.cost:.2f} added.")
+        print("\nHere is the inventory:")
+    for i, guitar in enumerate(guitars, 1):
+        vintage_string = " (vintage)" if guitar.is_vintage() else ""
+        print(f"Guitar {i}: {guitar.name:>20} ({guitar.year}), worth ${guitar.cost:10,.2f}{vintage_string}")
+        if __name__ == '__main__':
```
Comment (a-rosa) [inline prac_06/guitars.py:20]: This isn't supposed to be written here, it should be in the farthest left, also you can just call main function with main() only

File: prac_06/programming_language.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        print(python)
+        print("The dynamically typed languages are:")
+        for language in languages:
+          if language.is_dynamic():
+              print(language.name)
+
+              if __name__ == "__main__":
+                  run_tests()
+
+
```
Comment (a-rosa) [inline prac_06/programming_language.py:32]: I think it would be better to delete the run test function

**Pull request merged**